cmd/podman/start.go

```diff
@@ -96,6 +96,14 @@ func startCmd(c *cli.Context) error {
 				inputStream = nil
 			}
 
+			ctrState, err := ctr.State()
```

Comment:
how does the --attach param play here?  Up above it says you can't attach to multiple containers but it looks like you are here?  Is that parameter necessary anymore?

Reply:
@TomSweeneyRedHat Attach requiring a single container is checked above, when we verify arguments.

Reply:
This all takes place inside a large `if attach {`